drupal/files/settings.silta.php

```diff
@@ -74,4 +74,6 @@
 /**
  * Use our own services override.
  */
-$settings['container_yamls'][] = 'sites/default/silta.services.yml';
\ No newline at end of file
+if (PHP_SAPI !== 'cli') {
```

Comment:
@Rade333 could you please add a comment here that we skip the CLI because monolog interferes with drush's use of stdout?

Reply:
Sure, here you go https://github.com/wunderio/charts/pull/123